### ISSUE TITLE
clear linux with openvino with movidius myriad x

### ISSUE THREAD

**Issue by hasnulaxmy** [clearlinux/distribution#2274]
Right now, my system is integrated with movidius myriad –x VPU and using clear linux OS. I installed Openvino inside it. My question is how we want to [Run the Docker* Image for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_docker_linux.html) . It seems like we need to install hddl package or openvino package first but after I did some research, I only found hddl package for Ubuntu and centos.
 
So basically, how we want to run this command properly: “docker run -it --rm --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp <image_name>”

**Comment by tankienleong** [comment]
use --privileged -v /dev:/dev if you want to run myriad inside docker
